src/components/ae-fraction/ae-fraction.vue

```diff
@@ -0,0 +1,81 @@
+<template>
+  <span class="ae-fraction" :class="[ fill ]">
+    <sup>{{ numerator }}</sup>
+    &frasl;
+    <sub>{{ denominator }}</sub>
+  </span>
+</template>
+<script>
+export default {
+  name: 'ae-fraction',
+  props: {
+    /**
+     * Fill property changes the color state of the fraction component.
+     * Select something between: `primary, secondary, neutral, alternative, white, black`
+     */
+    fill: {
+      type: String,
+      validator: value => [
+        'primary',
+        'secondary',
+        'neutral',
+        'alternative',
+        'white',
+        'black',
+      ].includes(value),
+    },
+
+    /**
+     * The numerator property defines
+     * the upper number in the fraction.
+     */
+    numerator: {
+      type: [Number, String],
+      default: 0,
+    },
+
+    /**
+     * The numerator property defines
+     * the lower number in the fraction.
+     */
+    denominator: {
+      type: [Number, String],
+      default: 0,
+    },
+  },
+};
+</script>
+<style lang="scss" scoped>
+@import '../../styles/variables/colors';
+@import '../../styles/variables/typography';
+
+.ae-fraction {
+  display: inline-block;
+  font-family: $font-sans;
+  font-weight: 500;
+  font-style: normal;
+  font-stretch: normal;
+  line-height: 0;
+  word-spacing: -4px;
+  color: inherit;
```

Comment:
It isn't `inherit` by default?

Reply:
Auto is by default, not inherit.

Reply:
In with case, this difference has sense? Can't we avoid setting of `color` property?

Reply:
I removed, though would have liked it for this component to inherit parent color, but this works as well.